plugin/trino-sqlserver/src/main/java/io/trino/plugin/sqlserver/SqlServerClient.java

```diff
@@ -970,6 +983,11 @@ private static String singleQuote(String literal)
         return "\'" + literal + "\'";
     }
 
+    private static String doubleQuote(String literal)
+    {
+        return "\"" + literal + "\"";
```

Comment:
verify `literal` does not contain a double quote

Reply:
I'd even go as far as saying to create just a single method called `quoteObjectName` which does both the wrapping within double quotes + doubling the double quotes. No opportunity for caller to mess it up.